pkgs/applications/video/mpv/scripts/simple-mpv-webui.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub }:
```

Comment:
Personal preference that makes future diffs easier to review & read:

```suggestion
{ stdenv
, fetchFromGitHun
}:
```

Reply:
I used nixfmt to format this expressions, but sure I can change it!

Reply:
Applied and amended. Thanks for the suggestion!